dist/build.gradle

```diff
@@ -21,7 +21,6 @@ dependencies {
   compile project(':api')
   compile project(':core')
   compile project(':impl')
-  compile project(':management')
```

Comment:
Correct and indeed it is no longer needed in the current scope of unclustered GA.